UPGRADE.md

```diff
@@ -7,6 +7,14 @@ When upgrading Phragile to a newer version **always** run
 
 after updating the files in the root directory of your application.
 
+## From 2.0.0 release to 3.0.0
```

Comment:
This is probably fine although when previously using version 1.0 or 1.1, and then getting 3.0.0 (ie. skipping 2.0.0) `snapshots:migrate` will also convert snapshots! So there is no intermediate step needed (first update to 2.0.0, and then to 3.0.0). I am not sure if this is intended to allow such a "super upgrade", though.